light-curve/tests/light_curve_py/features/test_ptp_var.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+from numpy.testing import assert_allclose
+
+from light_curve.light_curve_py import PeakToPeakVar
+
+
+def test_ptpvar():
```

Comment:
I still don't really understand what this test tests. Should it work for any statistical feature, not this one only?